udapi/block/read/conllu.py

```diff
@@ -86,6 +105,19 @@ def process_document(self, document):
                 logging.debug('No next bundle to process')
                 break
 
+            # Check if all lines have correct number of data fields.
+            # Skip invalid bundle otherwise.
+            raw_bundle_check = True
+            for line in raw_bundle:
+                if re_comment_like.search(line) is not None:
+                    continue
+                if len(line.split('\t')) != len(self.node_attributes):
```

Comment:
Loading CoNLL-U is the speed-critical part of code. Here the split('\t') is done twice, so I am afraid it is slower than the previous version. It should be benchmarked, e.g. using the https://github.com/martinpopel/newtreex#benchmark (where "utreex" is an older nickname for this python-udapi). 
As for validity checking: ideally there should be a fast&robust way and slower&strict way of loading.